var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -177,15 +177,14 @@ def build_voronoi(self):
                         'compute_voronoi_atom.h')
 
     def build(self, spec, prefix):
-        for pkg in self.supported_packages:
```

Comment:
i.e. `voronoi` has `with working_dir('src/VORONOI'):`. So i guess you need to call those functions in place where they were called originally.